src/syft/lib/torch/return_types.py

```diff
@@ -0,0 +1,130 @@
+# stdlib
+import re
+from typing import Any
+from typing import Dict
+from typing import List
+
+# third party
+from packaging import version
+import torch
+
+# syft relative
+from ...generate_wrapper import GenerateWrapper
+from ...lib.util import full_name_with_name
+from ...proto.lib.torch.returntypes_pb2 import ReturnTypes as ReturnTypes_PB
+from ..torch.tensor_util import protobuf_tensor_deserializer
+from ..torch.tensor_util import protobuf_tensor_serializer
+
+# add a fake module "return_types" to torch
```

Comment:
Add a TODO and issue in GitHub here to also possibly work around this?

Reply:
agree